### ISSUE TITLE
something wrong

### ISSUE THREAD

**Issue by NiuPDJ** [AndyCao1125/SpikeSlicer#2]
Thank you very much for your work. I tried to reproduce this work using your code, but the accuracy is far from the results in the paper (2.9mm, 8.9°). The parameters selected are: 

GazeSCRNN-Events1750-FPTT-Backprop8-TWT20
--num_users 48
--data_preload
--gpus 0
-b 64
--fptt
--backprop_timesteps 8
--data_event_count 1750
--data_augment_affine
--data_augment_noise 0.1
--time_window_threshold 20000. 

Could you please tell me where the settings are wrong? Or are there any other tricks I can refer to during training?
Looking forward to your reply, thank you very much!

**Comment by NiuPDJ** [comment]
Sorry, I mistakenly asked a question from another project here. Please delete it.
